### DISCUSSION TITLE
add Notifications channels over MQTT

### DISCUSSION THREAD

**Discussion by superkalt** [frangoteam/FUXA#403] [Ideas]
how about regulating the complete notification only via mqtt?

In the meantime you can help yourself very well with nodered! So the requests would be done via Telegram or SMS.

later you could create your own notification center where all possible services could be integrated.

**Comment by unocelli** [comment]
good idea, is there an example or a library?

**Comment by superkalt** [comment]
mqtt is already installed

just push something to server "mqtt://superkalt:1883" with topic "-1001763254176"

[https://t.me/+LioC-zY5IalkOTU6](https://t.me/+LioC-zY5IalkOTU6)

**Comment by superkalt** [comment]
![image](https://user-images.githubusercontent.com/44277174/173442408-00730afc-d94b-4537-9afc-992598d4e5cf.png)

```
var _nachricht = msg.payload;
var _chatId = "-1001763254176";
msg.payload = {chatId: _chatId, type: 'message', content: _nachricht};
return msg;
```

**Comment by superkalt** [comment]
![image](https://user-images.githubusercontent.com/44277174/173446731-d0f330ac-72d4-4b68-a588-05c28c3dc79f.png)

**Comment by unocelli** [comment]
how do you send messages, via a webapi? a service?

**Comment by superkalt** [comment]
open telegram [https://t.me/+LioC-zY5IalkOTU6](https://t.me/+LioC-zY5IalkOTU6)

and just push something to server "mqtt://superkalt:1883" with topic "-1001763254176"

with a MQTT client - nothing more.
